src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html

```diff
@@ -182,10 +182,11 @@ <h5 class="group-title font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
                             <span>{{ 'artemisApp.examManagement.exerciseGroup.points' | artemisTranslate }}</span>
                         </th>
                         <th>
-                            <span>{{ 'artemisApp.exercise.bonusPoints' | artemisTranslate }}</span>
+                            <span>{{ 'artemisApp.exercise.bonus' | artemisTranslate }}</span>
                         </th>
                         <th>
-                            <span>{{ 'artemisApp.exercise.includedInOverallScore' | artemisTranslate }}</span>
+                            <!-- includedCompletely means includedInOverallScore here -->
```

Comment:
I don't get what this comment means to be honest

Reply:
I will just remove it